README.md

```diff
@@ -362,12 +367,109 @@ The `$SERVICE_BINDING_ROOT` environment variable **MUST NOT** be reset if it is
 
 If a `.spec.type` is set, the `type` entry in the binding `Secret` **MUST** be set to its value overriding any existing value.  If a `.spec.provider` is set, the `provider` entry in the binding `Secret` **MUST** be set to its value overriding any existing value.
 
+### Binding Status
+
 If the modification of the Application resource is completed successfully, the `Ready` condition status **MUST** be set to `True`.  If the modification of the Application resource is not completed successfully the `Ready` condition status **MUST NOT** be set to `True`.
 
 # Extensions
 
 Extensions are optional additions to the core specification as defined above.  Implementation and support of these specifications are not required in order for a platform to be considered compliant.  However, if the features addressed by these specifications are supported a platform **MUST** be in compliance with the specification that governs that feature.
 
+## Custom Projection
```

Comment:
Is there something that tells the "A" reconciler to stop the default projection when the custom projection path is triggered?

Reply:
good point, I will add that.

Reply:
added a new commit to address that.